website/content/en/docs/getting-started/_index.md

```diff
@@ -1,21 +1,86 @@
 ---
 title: Getting Started
-description: Trying ClusterLink for yourself
+description: This guide provides instructions for installing and configuring ClusterLink on Kubernetes. 
 weight: 20
 ---
 
+This guide will give you a quick start on installing and setting up the ClusterLink on a Kubernetes cluster.
+
 ## Prerequisites
 
-<!-- Are there any system requirements for using your project? What languages are supported (if any)? Do users need to already have any software or tools installed? -->
+Before you start, you must have access to a Kubernetes cluster. For example, you can set up a local environment using the [Kind](https://kind.sigs.k8s.io/) project.
 
 ## Installation
 
-<!-- Where can your user find your project code? How can they install it (binaries, installable package, build from source)? Are there multiple options/versions they can install and how should they choose the right one for them? -->
+1. To install ClusterLink on Linux or Mac, use the installation script:
+
+       $ curl -L <https://Clusterlink.net/install> | sh -
```

Comment:
I think adding backticks would make the code snipper copyable and also enable syntax highlighting.
If that works - please change throughout
````suggestion
```sh
curl -L <https://Clusterlink.net/install> | sh -
```
````